server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -69,7 +82,8 @@ private static void message(String msg, Opts opts) {
     boolean verbose = false;
   }
 
-  public static void main(String[] args) {
```

Comment:
The ZooZap main class is mentioned in the draft release notes for 2.1.0, and should be updated to reflect the keyword instead, since the main method is removed.

The assemble/bin/accumulo-cluster script tries to execute this as a main class, and will need to be updated to use the keyword executable, or it will need to have the main method re-added as an alternate entry point.

Reply:
Sorry, I meant to keep the main method. I will push a fix.

Reply:
Fixed in #2597